City-of-Pori/pori-kada
# PORI-24 General page styles

![screen shot 2017-10-26 at 17 00 12](https://user-images.githubusercontent.com/2276077/32057186-2ca2b48c-ba6f-11e7-8388-6104ea020c9c.png)

![screen shot 2017-10-26 at 17 00 47](https://user-images.githubusercontent.com/2276077/32057212-426b5ecc-ba6f-11e7-9123-b3f542885e27.png)


To test:

1. drush fra -y && drush cc all
2. Go to contexts and disable top_menu context
3. Edit a node such as Asuminen ja ympäristö. Remove all Topics so that they don't bother us at this point. Add the following html to body:

`<h2>Heading 2</h2><h3>Heading 3</h3><h4>Heading 4</h4><h5>Heading 5</h5><p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc nec lacinia erat. Integer in tortor a est facilisis sodales. Mauris quis finibus sem, sit amet malesuada odio. Phasellus iaculis ipsum turpis, at interdum leo malesuada in. In sagittis quam non augue ultricies, id pretium metus laoreet. Aenean quis ligula sit amet leo luctus porta. Nunc nibh lorem, ultrices in tincidunt sed, eleifend a lectus. In quam purus, gravida sed mi non, mattis pretium urna. Cras maximus velit risus, id lobortis nunc consequat vitae. Nunc tempus mi nec ultricies eleifend.&nbsp;<a href="https://google.com">Quisque mattis lacus justo, ut varius sapien suscipit eget.</a></p><p>Nunc convallis purus vitae finibus gravida. Donec blandit facilisis enim, in tincidunt arcu auctor quis. Mauris tristique magna quis felis efficitur efficitur. Sed ornare, justo sit amet varius dignissim, lorem nisl maximus massa, vitae tincidunt nisi nunc nec nunc. Duis eget arcu lorem. Quisque congue neque vitae mi ultricies, ut interdum magna consequat. Mauris varius nibh vitae scelerisque luctus. Fusce vel sodales urna. Mauris et magna eleifend enim condimentum auctor in id nisi. Sed vehicula dapibus mauris. Sed tempor tincidunt libero, euismod suscipit risus aliquam quis. Fusce tincidunt, urna quis auctor interdum, dolor lacus semper mauris, id convallis mauris libero vitae orci.</p>`

4. Save the content.
5. Go to the Asuminen ja ympäristö page and force refresh your browser. You should see correct fonts being used and nice spacing between headers and paragraphs. You can check the attached screenshot for what it should look like if in doubt. Also make sure that the desktop navigation when hovered works as in screenshot (the screen needs to be quite wide for this to happen)
